other.py

```diff
@@ -875,7 +875,7 @@ def forward(self, src):
         return output, None
 
     def forgetting_curve(self, t, s):
```

Comment:
This `forgetting_curve` is unused in the prediction of `GRU_P`.

Reply:
That is correct, it doesn't even have such function, only Transformer and RNN classes are modified